docker/ssl/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+nginx:
+  build: nginx-image
+  links:
+    - letsencrypt
+    - iri
+  environment:
+    - MY_DOMAIN_NAME=__DOMAIN__
+    - PORT=__PORT__
+  ports:
+    - "80:80"
+    - "443:443"
+  volumes_from:
+    - letsencrypt
+letsencrypt:
+  image: quay.io/letsencrypt/letsencrypt:latest
+  command:  bash -c "sleep 6 && certbot certonly --standalone --email "__EMAIL__" -d "__DOMAIN__" -n --agree-tos"
+  entrypoint: ""
+  volumes:
+    - /etc/letsencrypt
+    - /var/lib/letsencrypt
+  ports:
+    - "80"
+    - "443"
+  environment:
+    - TERM=xterm
+iri:
+  image: iotaledger/iri:v1.5.3
+  command: -p __PORT__
+  ports:
+    - "__PORT__:__PORT__"
```

Comment:
This will publish the plaintext IRI API port, do we want to do this? I believe that we should only expose Nginx ports: HTTP and HTTPS.
You can put the `nginx` and the `iri` containers on the same docker network / or linking them like you already did, and use the internal Docker DNS to resolve IRI's container's IP from `nginx` instead of `sed`ing `__APPLICATION_IP__`